openslides_backend/action/actions/user/user_scope_permission_check_mixin.py

```diff
@@ -69,15 +69,25 @@ def check_permissions_for_scope(self, instance: Dict[str, Any]) -> None:
         else:
             raise MissingPermission({OrganisationManagementLevel.CAN_MANAGE_USERS: 1})
 
-    def get_user_scope(self, id: int) -> Tuple[UserScope, int]:
+    def get_user_scope(
+        self, id: int, instance: Optional[Dict[str, Any]] = None
```

Comment:
If user is given, `id` is always `None`. It would be cleaner to make both arguments optional, so that in this case it is enough to call `get_user_scope(user=instance)`.

Reply:
Done